src/model.test.ts

```diff
@@ -1,113 +1,842 @@
+import * as faker from 'faker';
 import * as _ from 'lodash';
 import { Request } from 'express';
 import { HubApiModel } from '../src/model';
+import { PagingStream } from "../src/paging-stream";
 import { PassThrough, pipeline } from 'stream';
 import { promisify } from 'util';
-import axios from 'axios';
-jest.mock('axios');
+import { lookupDomain } from '@esri/hub-common';
+import { getOgcItemsStream } from './helpers/get-ogc-items-stream';
+
+jest.mock('@esri/hub-search');
+jest.mock('./helpers/get-ogc-items-stream');
+jest.mock('@esri/hub-common', () => ({
+  ...(jest.requireActual('@esri/hub-common') as object),
+  hubApiRequest: jest.fn(),
+  lookupDomain: jest.fn(),
+  fetchSiteModel: jest.fn()
+}));
 
 describe('HubApiModel', () => {
-  it('streams hub search ogc items', async () => {
-    const mockedResponse = {
-      data: {
-        features: [{
-          id: 'a123',
-          type: 'Feature',
-          properties: {
-            title: 'item-title'
-          }
-        }],
-        links: [
-          {
-            rel: 'prev'
-          }
-        ]
-      }
-    };
-    (axios.get as jest.Mock).mockResolvedValue(mockedResponse);
+  // this is just to make the type checker happy
+  const mockGetBatchStreams = getOgcItemsStream as unknown as jest.MockedFunction<typeof getOgcItemsStream>;
+  const mockLookupDomain = lookupDomain as unknown as jest.MockedFunction<typeof lookupDomain>;
+
+  beforeEach(() => {
+    mockGetBatchStreams.mockReset();
+    mockLookupDomain.mockResolvedValue({
+      "id": "374730",
+      "hostname": "download-test-qa-pre-a-hub.hubqa.arcgis.com",
```

Comment:
We should probably remove this site / id from this test since this is an open source project

Reply:
Good point! I have removed the id.